.gn

```diff
@@ -29,6 +29,14 @@ default_args = {
   pw_build_PIP_CONSTRAINTS = [ "//scripts/setup/constraints.txt" ]
   pw_build_PIP_REQUIREMENTS = [ "//scripts/setup/requirements.build.txt" ]
 
+  pw_build_LINK_DEPS = [
```

Comment:
Just wondering if this could be added as a dependency to pw_test instead of as a default

Reply:
That's a good idea, but I don't know how to do this. I tried a few "obvious" approaches and none of them worked. Do you have something specific in mind?

Reply:
@mbknust could you add a TODO for this here and then we can merge? it seems like how to achieve this is not immediately obvious however we should try to eventually have the dependency moved.

Reply:
> I am okay with just creating a TODO comment that points to a github issue
Would you prefer I create the issue before this is merged (so it can be referenced in the comment), or afterwards? This feels a bit chicken-and-egg.

Reply:
For future reference, @mbknust : just create the issue before the PR merges, put link to the issue in the code.